easybuild/easyblocks/generic/mesonninja.py

```diff
@@ -130,7 +130,7 @@ def configure_step(self, cmd_prefix=''):
         res = run_shell_cmd(cmd)
         return res.output
 
-    def build_step(self, verbose=False, path=None):
+    def build_step(self, path=None, *args, **kwargs):
```

Comment:
since `path` is not used:
```suggestion
    def build_step(self, *args, **kwargs):